### DISCUSSION TITLE
Exntending Twig - Changing content on an existing Kimai Page

### DISCUSSION THREAD

**Discussion by filippivarci** [kimai/kimai#5152] [Plugins]
I have developed a simple plugin for Kimai and I had no issues making a new page and creating what I need there.

However, I would like to modify existing rendered pages that are already on Kimai. For example in the core system we have templates/user/2fa.html.twig inside there is a block {% block form_body %} 

So in my plugin I created Resources/views/user/2fa.html.twig
and in this file I added:

{% extends user/2fa.html.twig %}
{% block form_body %} 
test
{% endblock %}

I thought that this would override the block form_body and show "test" on that page but I can't get it to work. Could anyone please let me know what I'm missing?

I tried {% extends @Kimai/user/2fa.html.twig %} or {% extends @App/user/2fa.html.twig %} and so on but with no luck. Maybe I'm missing how to load the twig from my plugin or the path is wrong. Any help would be appreciated.

**Comment by kevinpapst** [comment]
Please ask such questions in the Symfony world => they have a dedicated Slack community.

But I can tell you (for this case) its impossible. Plugins can never overwrite app templates, only the other way around works.

**Reply by filippivarci** [reply]
Thank you for the answer, that's all I needed to know. 

I used to work with Shopware (based on symfony) and it was possible there - simply by using a theme. So I was hoping it would be possible too with Kimai.
